### DISCUSSION TITLE
Hi :)

### DISCUSSION THREAD

**Discussion by Steiner-254** [Cyfrin/foundry-full-course-cu#1936] [General]
Passing greetings to anyone working day and night to up their blockchain skills :)

**Comment by EngrPips** [comment]
That's me, @Steiner-254. I hope I make it soon, lol.

**Reply by Steiner-254** [reply]
You got this, keep pushing :)

**Reply by EngrPips** [reply]
Indeed!

**Comment by Ifra001** [comment]
Its slow right now but hoping to make it through this journey!!

**Reply by EngrPips** [reply]
You can do it. You need to keep going. This is a marathon and not a sprint. So take breaks, restrategize, and crush it all!
